forge-gui/res/adventure/Shandalar/skin/ui_skin.json

```diff
@@ -395,6 +390,11 @@
 		"selection": "select"
 	}
 },
+"com.badlogic.gdx.scenes.scene2d.ui.TextTooltip$TextTooltipStyle": {
```

Comment:
this causes issues on android not working, try to remove it...
![SerializationException](https://media.discordapp.net/attachments/885197610862927933/959988470007345172/unknown.png)